### PR TITLE
feat(datastore): create connection with infodengue db & api for historico_alerta table

### PR DESCRIPTION
**Pull request by luabida**

### CODE REVIEW

File: src/datastore/models.py [hunk 1]
```diff
@@ -1 +1,43 @@
-# Create your models here.
+from django.db import models
+
+
+class HistoricoAlerta(models.Model):
```
Comment (luabida) [inline src/datastore/models.py:4]: @fccoelho please let me know if there are columns you think that should not be fetched from the infodengue's table

**Comment by luabida** [comment]
Usage:


image


```sh
curl -X 'GET' \
  'http://0.0.0.0:8042/api/datastore/historico_alerta/?start=2016-10-23&end=2023-10-23&page=1&per_page=100' \
  -H 'accept: application/json'
```

RESULT:


```json
{
  "items": [
    {
      "data_iniSE": "2023-07-23",
      "SE": 202330,
      "casos_est": 0,
      "casos_est_min": 0,
      "casos_est_max": 0,
      "casos": 0,
      "municipio_geocodigo": 5215801,
      "p_rt1": 0,
      "p_inc100k": 0,
      "Localidade_id": 0,
      "nivel": 1,
      "id": 521580120233019600,
      "versao_modelo": "2023-08-02",
      "municipio_nome": "Palmelo",
      "pop": 2381,
      "tempmin": 18.7142857142857,
      "umidmax": 56.6721522857143,
      "receptivo": 0,
      "transmissao": 0,
      "nivel_inc": 0,
      "umidmed": 55.1192077142857,
      "umidmin": 53.5662634285714,
      "tempmed": 19,
      "tempmax": 19.2857142857143,
      "casprov": null,
      "casprov_est": null,
      "casprov_est_min": null,
      "casprov_est_max": null,
      "casconf": null
    },
    [98 HIDDEN]
    {
      "data_iniSE": "2023-07-23",
      "SE": 202330,
      "casos_est": 0,
      "casos_est_min": 0,
      "casos_est_max": 0,
      "casos": 0,
      "municipio_geocodigo": 1303403,
      "p_rt1": 0,
      "p_inc100k": 0,
      "Localidade_id": 0,
      "nivel": 1,
      "id": 130340320233019570,
      "versao_modelo": "2023-08-02",
      "municipio_nome": "Parintins",
      "pop": 115363,
      "tempmin": 26.1428571428571,
      "umidmax": 75.5316891428571,
      "receptivo": 0,
      "transmissao": 0,
      "nivel_inc": 0,
      "umidmed": 60.3832342857143,
      "umidmin": 51.2777132857143,
      "tempmed": 29.275,
      "tempmax": 31.4285714285714,
      "casprov": null,
      "casprov_est": null,
      "casprov_est_min": null,
      "casprov_est_max": null,
      "casconf": null
    }
  ],
  "pagination": {
    "items": 100,
    "total_items": 1963517,
    "page": 1,
    "total_pages": 19636,
    "per_page": 100
  },
  "message": ""
}
```

File: src/datastore/api.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+from .models import HistoricoAlerta
+from .schema import HistoricoAlertaSchema, HistoricoAlertaFilterSchema
+from registry.pagination import PagesPagination
+
+
+router = Router()
+
+paginator = PagesPagination
+paginator.max_per_page = 100
```
Comment (luabida) [inline src/datastore/api.py:15]: I've manually set the max_per_page to 100 in this case, but more than that already feels too much for a single request

**Comment by fccoelho** [comment]
We should have tests that makes sure we can download multiple pages programmatically.

Also add examples to the documentation of how to download these data using `requests` or [`httpx`](https://www.python-httpx.org/) libraries

File: src/datastore/models.py (hunk 1 above)
Comment (fccoelho) [reply src/datastore/models.py:4]: All columns can be served

**Review by fccoelho — APPROVED** [review]

**Comment by luabida** [comment]
@fccoelho I'm trying to test the infodengue connection, but django is returning this error saying that this operation is not allowed on tests, do you know something about it?

```
django.test.testcases.DatabaseOperationForbidden: Database queries to 'Municipio' are not allowed in this test. Add 'Municipio' to datastore.tests.test_api.DatastoreAPITest.databases to ensure proper test isolation and silence this failure.
```

**Comment by fccoelho** [comment]
I am not familiar with this test isolation issue, maybe it is related to the fact that it is a test on an external database

**Comment by fccoelho** [comment]
Maybe this test should be written to bypass django testing framework. Just doing a request to the endpoints as an API user would do in a script, and like you did above with `curl`, but with pagination.

**Pull request merged**